backends/advanced/src/advanced_omi_backend/routers/api_router.py

```
@@ -24,6 +24,7 @@
     queue_router,
     system_router,
     user_router,
+    annotation_router,
```

Ruff (0.14.14)

[error] Redefinition of unused `annotation_router` from line 15: `annotation_router` redefined here

Remove definition: `annotation_router`

(F811)